cohort_4/playbook/activities.md

```diff
@@ -58,6 +61,31 @@ first floor with the elevators.
 
 TODO
 
+## Demos
+
+### Presenting your demo
+
+Demos are the hackers' chance to show off what they have been putting so much
+time and effort into for the past day.
+
+- Whenever someone wants to present they can volunteer to (no set presentation
+  order)
```

Comment:
This is a pretty passive way to say that the order of demos depends on who volunteers.

Perhaps reword this to be a bit more direct?